client/bin/start.js

```diff
@@ -222,6 +237,16 @@ async function main() {
       continue;
     }
 
+    if (parsingFlags && arg === "--transport" && i + 1 < args.length) {
```

Comment:
Could we `import { parseArgs } from "node:util";` as in server/src/index.ts to avoid implementing our own args parsers?

(and avoid awkward discrepancies, e.g. this cli needs --transport separate from its value, while the inspector server above accepts --transport=value)

Reply:
Ack, mind if I do this in a follow-up? Feel like it's a bit out of scope for this specific change given the pre-existing patterns here and I'd want to avoid opening a different can of worms in the same PR.

Reply:
Absolutely 👌